test/test_helper.rb

```diff
@@ -11,7 +11,6 @@
 require File.dirname(__FILE__) + '/../lib/identity_cache'
 
 DatabaseConnection.setup
-ActiveSupport::Cache::Store.instrument = true
```

Comment:
I think this is causing test failures.  E.g. `0 instead of 1 memcache operations were executed` I think is because it needs this instrumentation to count memcache operations